### PR TITLE
Add/Modify/Remove files for housekeeping

### PR DESCRIPTION
**Pull request by j-mes**
- Add Circle & remove Travis.
- Remove `.travis.yml`.
- Add `circle.yml`.
- Modify `origami.json` to have CircleCI.
- Modify `README.md` to have CircleCI shields & link.
- Fix linting in `test/viewport.test.js`.

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+machine:
+  node:
+    version: 4.2.2
```
Comment (rowanmanning) [inline circle.yml]: Does this need to be explicit, or could it just be `4`?

File: origami.json [hunk 2]
```diff
@@ -11,5 +11,8 @@
 		"optional": [
 			"matchmedia"
 		]
+	},
+	"ci": {
+		"circle": "https://circleci.com/api/v1/project/Financial-Times/o-viewport"
```
Comment (rowanmanning) [inline origami.json]: The [Origami JSON spec](http://origami.ft.com/docs/syntax/origamijson/#format) doesn't mention a `circle` property. Maybe we should open a PR to update the spec page and allow it?

**Comment by rowanmanning** [comment]
Couple of comments, otherwise looking good 👍

File: origami.json (hunk 2 above)
Comment (j-mes) [inline origami.json]: Adding this later this afternoon!

File: circle.yml (hunk 1 above)
Comment (j-mes) [inline circle.yml]: I'll try this with 4 and see what happens when it goes through CircleCI

**Pull request merged**